config/kubernetes-sigs/sig-api-machinery/teams.yaml

```diff
@@ -59,27 +59,15 @@ teams:
   controller-tools-admins:
     description: admin access to controller-tools
     members:
-    - DirectXMan12
-    - droot
-    - pwittrock
-    - seans3
```

Comment:
Not sure about @seans3, it doesn't show up in any commits in https://github.com/kubernetes-sigs/controller-tools/commits/master/OWNERS_ALIASES? This might precede my time in this repo